Examples/DotNetCoreBuild/test/TestDirectories/testDirectories.dsc

```diff
@@ -56,7 +56,10 @@ export const writeHardLinkInSharedOpaqueDirectoryIsAllowed = !Context.isWindowsO
     linkFileIntoDirectory(f`testDirectories.dsc`, sod);
 
     // hardlink an output file
-    const outFile = Transformer.writeFile(p`${Context.getNewOutputDirectory("write-file")}/writefile.txt`, "hi");
+    const outFile = Transformer.writeAlLText({

```

Comment:
>writeAlLText [](start = 32, length = 12)

typo?

Reply:
> > writeAlLText [](start = 32, length = 12)
> 
> typo?
> If yes, how could this test have passed?

It was a typo and it didn't pass /partB validation.